flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
how does one use this on a Raspberry Pi or Asahi Linux?

Reply:
Well, at least something than nothing. Maybe I will support other platforms later.